foxbot_core_config.h

```diff
@@ -129,6 +129,15 @@ float angular_velocity_est;
 float yaw_est;
 unsigned long odom_prev_time;
 
+/* Define JointState variables */
+char *joint_name[2] = {"wheel_right_joint", "wheel_left_joint"};
+float pos[2];
+float vel[2];
+float eff[2];
```

Comment:
joints_pos, joints_vel and joints_eff would avoid confusion with other variables